### PR TITLE
Joslin dcunha

### PR DESCRIPTION
**Pull request by joslindcunha**
This Pull Request resolves Issue #1 

Description:
In this Pull Request we have done the following additions/edits for the code:

### CODE REVIEW

**Comment by joslindcunha** [comment]
done

**Comment by salman-bhai** [comment]
Still shows conflicts @joslindcunha

**Comment by salman-bhai** [comment]
Check it on Github !

File: PARTICIPANTS/joslin1.txt.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+NAME: JOSLIN ANISETH DCUNHA
```
Comment (salman-bhai) [inline PARTICIPANTS/joslin1.txt.txt]: Delete this file!

**Comment by salman-bhai** [comment]
Please delete all other files. You're PR should have just one file which is the `.txt` file

**Comment by salman-bhai** [comment]
SHould this PR be closed?

**Comment by salman-bhai** [comment]
Any updates here? Or should we close the Pull Request?

**Pull request closed without merging**